pkg/kv/kvserver/client_protectedts_test.go

```diff
@@ -180,6 +180,15 @@ ORDER BY raw_start_key ASC LIMIT 1`)
 		Target:    ptpb.MakeSchemaObjectsTarget([]descpb.ID{getTableID()}),
 	}
 	require.NoError(t, ptsWithDB.Protect(ctx, &ptsRec))
+	// Verify that the record did indeed make its way down into KV where the
+	// replica can read it from.
+	ptsReader := tc.GetFirstStoreFromServer(t, 0).GetStoreConfig().ProtectedTimestampReader
+	require.NoError(
+		t,
+		ptutil.TestingVerifyProtectionTimestampExistsOnSpans(
```

Comment:
Could you rename this helper to something that captures that there's a SucceedsSoon block internally? WaitUntil...? There's one usage that wraps this helper around a SucceedsSoon block as well, so it's confused at least one other person.